components/shared_code/src/shared_data_model/sources/junit.py

```diff
@@ -19,6 +19,7 @@
     name="test",
     attributes=[
         EntityAttribute(name="Class name"),
+        EntityAttribute(name="Host name"),
```

Comment:
Can this be an optional, i.e. column hidden when none of the entities have it?

Reply:
Unfortunately not, at the moment.